.github/workflows/reusable-build-test-release.yml

```diff
@@ -909,6 +909,7 @@ jobs:
       k8s-manifests-branch: ${{ steps.test-setup.outputs.k8s-manifests-branch }}
       argo-namespace: ${{ steps.test-setup.outputs.argo-namespace }}
       addon-name: ${{ steps.test-setup.outputs.addon-name }}
+      addon-folder-name: ${{ steps.test-setup.outputs.addon-folder-name }}
```

Comment:
I think we could get addon-folder-name later on from here and pass it to btool:
https://github.com/splunk/ta-automation-app-of-apps/blob/main/src/workflows/argo-workflow-templates/components/resources/test-addon.yaml#L133

That would spare us additional parameters and some code changes.